src/eip.rs

```diff
@@ -15,14 +17,20 @@ const EIP_POD_UID_TAG: &str = "eip.aws.materialize.com/pod_uid";
 pub(crate) async fn allocate_address(
     ec2_client: &Ec2Client,
     pod_uid: String,
+    default_tags: &HashMap<String, String>,
 ) -> Result<AllocateAddressOutput, SdkError<AllocateAddressError>> {
+    let mut tags: Vec<Tag> = default_tags
+        .into_iter()
```

Comment:
the GitHub Actions lint here to use `iter()` instead of `into_iter()` ain't half bad.

Reply:
done